decidim-core/lib/decidim/core/api/user_type.rb

```diff
@@ -13,11 +13,15 @@ module Decidim
     field :name, !types.String, "The user's name"
 
     field :avatarUrl, !types.String, "The user's avatar url" do
-      resolve ->(obj, _args, _ctx) { obj.avatar.url }
+      resolve ->(obj, _args, _ctx) { obj.deleted? ? ActionController::Base.helpers.asset_path("decidim/default-avatar.svg") : obj.avatar.url }
```

Comment:
This smells. Shouldn't we change the avatar to this when deleting the user? This way you don't need to check this value in the API level, you just change it when deleting the user.